### ISSUE TITLE
Do the same with functions ?

### ISSUE THREAD

**Issue by ronnyremesnik** [suusan2go/kotlin-fill-class#89]
Can we do the same completion but with functions ?

**Comment by oboenikui** [comment]
Yes, it is possible. If there is a problem with function completion, could you please provide more context about the problem you are facing with?
